jobs/JGDAS_ATMOS_VERFRAD

```diff
@@ -6,40 +6,34 @@
 source "${HOMEgfs}/ush/preamble.sh"
 source "${HOMEgfs}/ush/jjob_header.sh" -e "verfrad" -c "base verfrad"
 
-export COMPONENT="atmos"
-
 export RADMON_SUFFIX=${RADMON_SUFFIX:-${RUN}}
 export CYCLE_INTERVAL=${CYCLE_INTERVAL:-6}
+export VERBOSE=${VERBOSE:-YES}
```

Comment:
`CYCLE_INTERVA` is the same as `assim_freq`

Reply:
Since I see `CYCLE_INTERVAL` within scripts in the GSI-Monitor package I have left it being set in the script but have changed it to use `assim_freq`.